From 2995ba7157147d163ea6ae036b4e8b9cc394ab88 Mon Sep 17 00:00:00 2001
From: Sebastiano Barezzi <barezzisebastiano@gmail.com>
Date: Thu, 25 Mar 2021 14:53:13 +0100
Subject: [PATCH 136/178] input: touchscreen: synaptics_tcm_*: Rename I2C and
 SPI configs

* They get defined twice due to having 2 synaptics_tcm Kconfigs

Change-Id: I71b9df12c7c23714b47c213cbfeed754c9e49c73
---
 drivers/input/touchscreen/synaptics_tcm_d2t/Kconfig  | 6 +++---
 drivers/input/touchscreen/synaptics_tcm_d2t/Makefile | 4 ++--
 drivers/input/touchscreen/synaptics_tcm_f7a/Kconfig  | 6 +++---
 drivers/input/touchscreen/synaptics_tcm_f7a/Makefile | 4 ++--
 4 files changed, 10 insertions(+), 10 deletions(-)

diff --git a/drivers/input/touchscreen/synaptics_tcm_d2t/Kconfig b/drivers/input/touchscreen/synaptics_tcm_d2t/Kconfig
index 13d0be2e7ac9..cec7a4c837c3 100644
--- a/drivers/input/touchscreen/synaptics_tcm_d2t/Kconfig
+++ b/drivers/input/touchscreen/synaptics_tcm_d2t/Kconfig
@@ -13,12 +13,12 @@ menuconfig TOUCHSCREEN_SYNAPTICS_TCM_D2T
 if TOUCHSCREEN_SYNAPTICS_TCM_D2T
 
 choice
-	default TOUCHSCREEN_SYNAPTICS_TCM_I2C
+	default TOUCHSCREEN_SYNAPTICS_TCM_I2C_D2T
 	prompt "Synaptics TCM bus module"
-config TOUCHSCREEN_SYNAPTICS_TCM_I2C
+config TOUCHSCREEN_SYNAPTICS_TCM_I2C_D2T
 	bool "I2C"
 	depends on I2C
-config TOUCHSCREEN_SYNAPTICS_TCM_SPI
+config TOUCHSCREEN_SYNAPTICS_TCM_SPI_D2T
 	bool "SPI"
 	depends on SPI_MASTER
 endchoice
diff --git a/drivers/input/touchscreen/synaptics_tcm_d2t/Makefile b/drivers/input/touchscreen/synaptics_tcm_d2t/Makefile
index de2953771690..c5fe42826525 100644
--- a/drivers/input/touchscreen/synaptics_tcm_d2t/Makefile
+++ b/drivers/input/touchscreen/synaptics_tcm_d2t/Makefile
@@ -4,8 +4,8 @@
 
 # Each configuration option enables a list of files.
 
-obj-$(CONFIG_TOUCHSCREEN_SYNAPTICS_TCM_SPI) += synaptics_tcm_spi.o
-obj-$(CONFIG_TOUCHSCREEN_SYNAPTICS_TCM_I2C) += synaptics_tcm_i2c.o
+obj-$(CONFIG_TOUCHSCREEN_SYNAPTICS_TCM_SPI_D2T) += synaptics_tcm_spi.o
+obj-$(CONFIG_TOUCHSCREEN_SYNAPTICS_TCM_I2C_D2T) += synaptics_tcm_i2c.o
 obj-$(CONFIG_TOUCHSCREEN_SYNAPTICS_TCM_CORE) += synaptics_tcm_core.o
 obj-$(CONFIG_TOUCHSCREEN_SYNAPTICS_TCM_TOUCH) += synaptics_tcm_touch.o
 obj-$(CONFIG_TOUCHSCREEN_SYNAPTICS_TCM_DEVICE) += synaptics_tcm_device.o
diff --git a/drivers/input/touchscreen/synaptics_tcm_f7a/Kconfig b/drivers/input/touchscreen/synaptics_tcm_f7a/Kconfig
index 5ff2f994e4c8..a371a587aa0b 100644
--- a/drivers/input/touchscreen/synaptics_tcm_f7a/Kconfig
+++ b/drivers/input/touchscreen/synaptics_tcm_f7a/Kconfig
@@ -12,12 +12,12 @@ menuconfig TOUCHSCREEN_SYNAPTICS_TCM_F7A
 if TOUCHSCREEN_SYNAPTICS_TCM_F7A
 
 choice
-	default TOUCHSCREEN_SYNAPTICS_TCM_I2C
+	default TOUCHSCREEN_SYNAPTICS_TCM_I2C_F7A
 	prompt "Synaptics TCM bus module"
-config TOUCHSCREEN_SYNAPTICS_TCM_I2C
+config TOUCHSCREEN_SYNAPTICS_TCM_I2C_F7A
 	bool "I2C"
 	depends on I2C
-config TOUCHSCREEN_SYNAPTICS_TCM_SPI
+config TOUCHSCREEN_SYNAPTICS_TCM_SPI_F7A
 	bool "SPI"
 	depends on SPI_MASTER
 endchoice
diff --git a/drivers/input/touchscreen/synaptics_tcm_f7a/Makefile b/drivers/input/touchscreen/synaptics_tcm_f7a/Makefile
index 30db3b56516a..89bfb7744ed9 100644
--- a/drivers/input/touchscreen/synaptics_tcm_f7a/Makefile
+++ b/drivers/input/touchscreen/synaptics_tcm_f7a/Makefile
@@ -4,8 +4,8 @@
 
 # Each configuration option enables a list of files.
 
-obj-$(CONFIG_TOUCHSCREEN_SYNAPTICS_TCM_SPI) += synaptics_tcm_spi.o
-obj-$(CONFIG_TOUCHSCREEN_SYNAPTICS_TCM_I2C) += synaptics_tcm_i2c.o
+obj-$(CONFIG_TOUCHSCREEN_SYNAPTICS_TCM_SPI_F7A) += synaptics_tcm_spi.o
+obj-$(CONFIG_TOUCHSCREEN_SYNAPTICS_TCM_I2C_F7A) += synaptics_tcm_i2c.o
 obj-$(CONFIG_TOUCHSCREEN_SYNAPTICS_TCM_CORE) += synaptics_tcm_core.o
 obj-$(CONFIG_TOUCHSCREEN_SYNAPTICS_TCM_TOUCH) += synaptics_tcm_touch.o
 obj-$(CONFIG_TOUCHSCREEN_SYNAPTICS_TCM_DEVICE) += synaptics_tcm_device.o
-- 
2.49.0

